spec/datadog/core/configuration/option_spec.rb

```diff
@@ -34,6 +34,12 @@
   let(:setter_value) { double('setter_value') }
   let(:context) { double('configuration object') }
 
+  before do
+    allow(Datadog::Core::Utils::SafeDup).to receive(:frozen_or_dup) do |args, _block|
+      args
+    end
+  end
```

Comment:
Minor: May be worth leaving here a comment (as well as on `options_spec`) to explain why this is needed.